core/src/main/kotlin/com/ritense/valtimo/CamundaWhitelistedBeansPlugin.kt

```diff
@@ -0,0 +1,43 @@
+/*
+ *  Copyright 2015-2024 Ritense BV, the Netherlands.
+ *
+ *  Licensed under EUPL, Version 1.2 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  https://joinup.ec.europa.eu/collection/eupl/eupl-text-eupl-12
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" basis,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package com.ritense.valtimo
+
+import mu.KotlinLogging
+import org.camunda.bpm.engine.impl.cfg.AbstractProcessEnginePlugin
+import org.camunda.bpm.engine.impl.cfg.ProcessEngineConfigurationImpl
+import org.camunda.bpm.engine.spring.SpringExpressionManager
+import org.springframework.context.ApplicationContext
+
+class CamundaWhitelistedBeansPlugin(
```

Comment:
What was the reason behind splitting this off? To keep more focus to the plugin?

Reply:
Its to fix a bug. By default, Valtimo doesn't do authorization on service tasks because those are always executed by the system. But the bug is; when you have `bean-whitelisting` set to false, suddenly this doesn't work anymore and you get authorization errors on service-tasks.

Reply:
Makes sense, thanks for the explanation!